module-audio/Audio/equalizer/Equalizer.cpp

```diff
@@ -21,6 +21,9 @@ namespace audio::equalizer
         if ((Q < qMinValue || Q > qMaxValue) && filter != FilterType::None) {
             throw std::invalid_argument("Q out of range");
         }
+        if (samplerate == 0 && filter != FilterType::None) {
+            throw std::invalid_argument("Sample rate has to be greater than zero");
+        }
```

Comment:
what with the case, when the filter is set to none? :trollface:

Reply:
Actually this shouldn't be a problem. The division by zero will happen, that's true, so omega value will be incorrect, but when filter is set to none, that value is not used anywhere.

Reply:
do we catch this division by zero anywhere? ( except in test :joy: )

Reply:
After quick analysis - I don't think so :P None of this exceptions seem to be caught anywhere...